client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/package-row.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { map, uniq } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import ItemRow, { Header as ItemRowHeader } from './item-row';
+import TextField from 'woocommerce/woocommerce-services/components/text-field';
+import Dropdown from 'woocommerce/woocommerce-services/components/dropdown';
+import Checkbox from 'woocommerce/woocommerce-services/components/checkbox';
+import {
+	setContentsType,
+	setContentsExplanation,
+	setRestrictionType,
+	setRestrictionExplanation,
+	setAbandonOnNonDelivery,
+	setITN,
+} from 'woocommerce/woocommerce-services/state/shipping-label/actions';
+import {
+	getShippingLabel,
+	isLoaded,
+	getFormErrors,
+} from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
+
+const PackageRow = ( props ) => {
+	const {
+		siteId,
+		orderId,
+		errors,
+		packageId,
+		translate,
+		contentsType,
+		contentsExplanation,
+		restrictionType,
+		restrictionExplanation,
+		abandonOnNonDelivery,
+		itn,
+		items,
+	} = props;
+	const abandonHandler = () => props.setAbandonOnNonDelivery( ! abandonOnNonDelivery );
+
+	return <div className="customs-step__package">
+		<label htmlFor={ packageId + '_abandonOnNonDelivery' }>
+			<Checkbox
+				id={ packageId + '_abandonOnNonDelivery' }
+				checked={ ! abandonOnNonDelivery }
+				onChange={ abandonHandler } />
```

Comment:
When this box is checked, the server expects a phone number to be entered for the origin address. However, our form does not mark it as required. Can we make sure to mark the phone number in the origin as required if the 'return to sender' box is checked?

Here's the error I got from the server when I tried it out:

> Error purchasing label 177: Unable to complete shipment purchase: Missing but required: return_address.phone when a customs form is necessary.

Reply:
Oh, nevermind - an origin phone number is _always_ necessary for any shipments that require a customs form. So, we always need to make sure it's required in our origin step when a customs form is required.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/24974/commits/b9062efdbf8758197167f1da8f34b33992e3d083

The origin phone is only required if the Customs form is required. So, if you go to the Destination address step and change it from domestic to a foreign country, it will make the Origin step be erroneous. That's a bit of an akward interaction but I think it's fine.